GVFS/GVFS.Installer.Mac/scripts/postinstall

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
 loadCmd="sudo launchctl load -w /Library/LaunchDaemons/org.vfsforgit.prjfs.PrjFSKextLogDaemon.plist"
 echo "Loading PrjFSKextLogDaemon: '$loadCmd'..."
-eval $loadCmd || exit 1
\ No newline at end of file
+eval $loadCmd || exit 1
+
+# Load Service in all active User sessions
+servicePlist="/Library/LaunchAgents/org.vfsforgit.service.plist"
```

Comment:
Gets the `loginwindow` `pid` and `uid` for each logged in user. Then use `launchctl bsexec` to auto load the Service for each user.

Reply:
This is good information, can we put this into the comment in the shell script? When it comes to shell scripts, I'd rather have the behavior clearly spelled out rather than trying to infer it.